x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -108,6 +109,9 @@ interface Props {
   onFilterAdded?: () => void;
 }
 
+export const disableHoverActions = (timelineId: string | undefined): boolean =>
```

Comment:
any benefit to wrapping this in a `useCallback`? It would just be a dependency on `timelineId`?